operator/bundle/openshift/manifests/loki-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
     service.beta.openshift.io/serving-cert-secret-name: loki-operator-metrics
   creationTimestamp: null
   labels:
-    app.kubernetes.io/component: metrics
```

Comment:
Is removing this label intentional? I'm not sure if we're using it on the ServiceMonitor instance ... I think it would be a good idea to keep the label to differentiate from the webhook service.

Reply:
Good catch! We see this issue only in the community-openshift and openshift bundles because of a patch that was resetting the base manifest labels. However the reset seems to be effective only with kustomize v5.